.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You need to add a new empty line at the end of the document in each file. Ask about it ChatGPT or google it for more info.